packages/utils/viem/isAbortError.ts

```diff
@@ -0,0 +1,3 @@
+export function isAbortError(e: any) {
```

Comment:
is not the same as the isAbortError from abortControl?

Reply:
ah it's viem error
how about we use `TimeoutError` from viem? It might be breaking if they change the message
but it will require utils to have viem deps

Reply:
Yes it's from viem. At first I tried with `TimeoutError` but actually viem is overriding the error type. For example if you make the call by `contract.simulate` then it'll throw an Error with name "ContractExecutionError", instead of the `TimeoutError`

They're not handling the abort error very well

Reply:
I guess you need to `walk()` the error if they wrapped the error in the details.
can we change the name, seems confusing if we have 2 same function name here

Reply:
ok interesting. Lemme check how their `walk` works